examples/recipes/llama/pretrain_llama3_8b.py

```diff
@@ -61,6 +61,7 @@
 from megatron.bridge.recipes.llama import llama3_8b_pretrain_config as pretrain_config
 from megatron.bridge.training.config import ConfigContainer
 from megatron.bridge.training.gpt_step import forward_step
+from megatron.bridge.training.post_training.checkpointing import has_modelopt_state
```

Comment:
I would duplicate this script in quantization folder, not use this one.
If i was an external user, I would go to examples/quantization for this.